accelerator/roles/intel/vars/main.yml

```diff
@@ -30,6 +30,7 @@ intel_habana_packages:
   - habanalabs-rdma-core
   - habanalabs-thunk
   - habanatools
+  - habanalabs-firmware-odm
```

Comment:
As discussed, please remove all the habanalabs-firmware-odm package installation changes

Reply:
Done